src/main/resources/META-INF/plugin.xml

```diff
@@ -64,7 +64,7 @@
 
     <change-notes><![CDATA[
       <ul>
-        <li>8.4 - Allow marking issues existing on SonarQube/SonarCloud as resolved. Support near real-time synchronization of Security Hotspots with SonarQube. Support syntax highlighting in rule configuration. 5 new rules and 1 quick fix for JavaScript. 2 new rules for C++. Bug fixes, fewer FPs and improvements for many languages.</li>
+        <li>8.4 - Allow marking issues existing on SonarQube/SonarCloud as resolved. Support near real-time synchronization of Security Hotspots with SonarQube. Support syntax highlighting in rule configuration. 5 new rules and 1 quick fix for JavaScript. 2 new rules for C++. 1 new rule for C# in Rider. Bug fixes, fewer FPs and improvements for many languages.</li>
```

Comment:
We could maybe mention the support for C# 7? I forgot to add it when I upgraded O#.

Reply:
Done.